templates/_linux.yaml

```diff
@@ -32,6 +33,11 @@ objects:
     template:
       spec:
         domain:
+{%- if item.iothreads == true %} {{''}}
+          ioThreadsPolicy: shared
+{%- elif item.iothreads == false or item.iothreads is not defined %}
```

Comment:
```suggestion
```

You can drop the line, it is not necessary.

Reply:
it's just double check - "never trust data input". In a case if we choose something strange in playbook.

Reply:
But the else section gives you nothing - no error, no warning - and is simply dead code.